zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -4,19 +4,23 @@ use std::collections::HashMap;
 
 use lazy_static::lazy_static;
 
-use zebra_chain::{amount::COIN, block::Height, parameters::Network};
+use zebra_chain::{
+    amount::COIN,
+    block::{Height, HeightDiff},
+    parameters::Network,
+};
 
 /// An initial period from Genesis to this Height where the block subsidy is gradually incremented. [What is slow-start mining][slow-mining]
 ///
 /// [slow-mining]: https://z.cash/support/faq/#what-is-slow-start-mining
-pub const SLOW_START_INTERVAL: Height = Height(20_000);
+pub const SLOW_START_INTERVAL: HeightDiff = 20_000;
```

Comment:
This seems more like a Height, in practice it's the maximum height that the slow start was active until.

it's never used as a difference or a repeated interval.

Reply:
155ce35d1d